docs/components/page-higher-order/withDocumentationElements.js

```diff
@@ -7,22 +7,24 @@ import DocumentationPage from '~/components/DocumentationPage';
 import { SluggerContext } from '~/components/page-higher-order/withSlugger';
 import * as components from '~/common/translate-markdown';
 
-export default meta =>
-  withRouter(
-    class DocumentationPageHOC extends React.Component {
-      render() {
-        const { router } = this.props;
-        return (
-          <DocumentationPage
-            title={meta.title}
-            url={router}
-            asPath={router.asPath}
-            sourceCodeUrl={meta.sourceCodeUrl}>
-            <SluggerContext.Provider value={new GithubSlugger()}>
-              <MDXProvider components={components}>{this.props.children}</MDXProvider>
-            </SluggerContext.Provider>
-          </DocumentationPage>
-        );
-      }
-    }
-  );
+const withDocumentationElements = (meta) => {
+  const DocumentationElementsHOC = withRouter((props) => {
```

Comment:
This might cause another anonymous component, I couldn't successfully reproduce the warning in the console.